e2e/e2e_test.go

```diff
@@ -301,17 +276,20 @@ var _ = Describe("e2e", func() {
 
 		Context("with classic remediation config", func() {
 
+			var nodeUnhealthyTime time.Time
+
```

Comment:
NIT: I am not sure that we need to define/declare of _nodeUnhealthyTime_ variable, it could be just defined in line 287 (`nodeUnhealthyTime :=`)